packages/benchmarks/lit-html/kitchen-sink/index.ts

```diff
@@ -114,6 +114,7 @@ const generateData = (
       'border-left-width': '0px',
       'border-right-width': '0px',
     },
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Seems like we need a lot of exceptions to `@typescript-eslint/no-empty-function`, and I think they're all valid. I think we should probably disable this rule.